Source/Libraries/CorruptCore/CorruptCore.cs

```diff
@@ -232,7 +232,6 @@ public static bool EmulatorOsdDisabled
         private static bool IsStandaloneUI;
         private static bool IsEmulatorSide;
 
-        private static EventHandler CorruptCoreExiting;
```

Comment:
I'm on mobile so I can't check but was this previously public and made private due to it not being referenced outside of this class? I'm fairly sure some of the incomplete plugins use this.

Reply:
I've restored `CorruptCoreExiting` and ignored the compiler warning.